packages/broker/src/plugins/operator/ContractFacade.ts

```diff
@@ -70,20 +70,25 @@ export interface SponsorshipResult {
     operatorCount: number
 }
 
+export type ExtendedOperatorServiceConfig = OperatorServiceConfig & {
```

Comment:
These values are used only in the `getEarningsOf`, and that method is used only from `checkOperatorValueBreach` function. 

Maybe it would make sense to give these as parameters to the `checkOperatorValueBreach` function? That function is used in `OperatorPlugin`, i.e. it can easily plug in the correct values from the `OperatorPluginConfig` object. The `checkOperatorValueBreach` function would then pass the values to `ContractFacade#getEarningsOf` when it calls it.

Reply:
Sounds better yeah, will refactor

Reply:
Fixed